DOL.WHD.Section14c.Web/src/modules/services/apiService.js

```diff
@@ -164,6 +164,27 @@ module.exports = function(ngModule) {
       return d.promise;
     };
 
+    this.checkPasswordComplexity = function(value) {
+      let url = _env.api_url + '/api/Account/PasswordComplexityCheck';
+      let d = $q.defer();
+
+      $http({
+        method: 'POST',
+        url: url,
+        headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+        data: value
+      }).then(
+        function successCallback(data) {
+          d.resolve(data);
+        },
+        function errorCallback(error) {
+          d.reject(error);
+        }
+      );
+
+      return d.promise;
```

Comment:
Just curious: since `$http(...)` returns a `$q` deferred/promise, could this be rewritten as:

```js
const url = _env.api_url + '/api/Account/PasswordComplexityCheck';

return $http({
  method: 'POST',
  url: url,
  headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
  data: value
});
```